README.adoc

```diff
@@ -6,6 +6,80 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Clusters
+
+In a new terminal, execute the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/host-operator
+$ minishift start --profile host
+$ make up-local
+```
+
+In another new terminal, execute the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/member-operator
+$ minishift start --profile member
+$ make up-local
+^C
+$ make add-member-to-host
+$ make add-host-to-member
+$ make up-local
+```
+
+Now that we are setup, we can add resources and watch the logs on the member-operator! To do so, open a new terminal and run the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/host-operator
+$ oc project toolchain-host-operator
+$ oc apply -f '/path/to/masteruserrecord.yaml'
+$ cd $GOPATH/github.com/codeready-toolchain/member-operator
+$ oc project toolchain-member-operator
+$ oc apply -f '/path/to/useraccount.yaml'
+```
+
+**Example masteruserrecord.yaml:**
+```
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: MasterUserRecord
+metadata:
+  name: example
+spec:
+  disabled: false
+  deprovisioned: false
+  userID: 86505192-a386-11e9-ad56-525400ad2b23
+  userAccounts:
+  - targetCluster: member-192-168-42-61-8443
```

Comment:
btw, just a detail - it would be good to add a description to this field as well - that the `targetCluster` should contain the correct IP address of the member cluster

Reply:
@MatousJobanek Good point! Added a note to this section to address that! Thanks!